apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -104,16 +104,16 @@ init_per_group(contracts, Config) ->
     ct:pal("ToMine ~p\n", [ToMine]),
     [ aecore_suite_utils:mine_key_blocks(NodeName, ToMine) || ToMine > 0 ],
 
-    %% Prepare accounts, Alice, Bert, Carl and Diana.
+    %% Prepare accounts, Alice, Bert, Carl and Diana with balance of 10T.
 
-    StartAmt = 250000,
+    StartAmt = 10*1000*1000000,                 %That is a lot of tokens!
     {APubkey, APrivkey, STx1} = new_account(StartAmt),
     {BPubkey, BPrivkey, STx2} = new_account(StartAmt),
     {CPubkey, CPrivkey, STx3} = new_account(StartAmt),
     {DPubkey, DPrivkey, STx4} = new_account(StartAmt),
 
     {ok, _} = aecore_suite_utils:mine_blocks_until_txs_on_chain(
-                                    NodeName, [STx1, STx2, STx3, STx4], 5),
```

Comment:
That is over the top ;-) 
There is a 1/50 chance you need 3 blocks, 1/50^2 for 4, etc...